src/Manifest-Resources-Tests/MFClassA.class.st

```diff
@@ -10,5 +10,5 @@ Class {
 { #category : #'as yet unclassified' }
 MFClassA >> method [
 	|foo|
-	self halt.
+	1+1.
```

Comment:
Why not just leaving an empty method returning `self`?

Reply:
I don't know. Do you want me to commit an empty method instead?

Reply:
An empty method would indeed be better.
Moreover, putting a message saying why that method is empty even better.

Reply:
Imagine in a couple of months the person that will read that 1+1 method that does not return and think it's a bug.

Reply:
Done 1227d92